tools/drdeploy.c

```diff
@@ -1529,7 +1606,7 @@ _tmain(int argc, TCHAR *targv[])
             die();
         for (j = 0; j < num_clients; j++) {
             if (!register_client(process, 0, global, dr_platform, client_ids[j],
-                                 client_paths[j], client_options[j]))
+                                 client_paths[j], alt_bitwidth[j], client_options[j]))
```

Comment:
Also, can you clarify to me why client_paths[j] and alt_bitwidth[j] can use the same indexes?